index.html

```diff
@@ -0,0 +1,107 @@
+            width="150px" height="50px"
+            alt="Youtube Logo">
+        </a>
+        <div class="search-container">
+            <input type="text" placeholder="Search">
+            <button type="submit"><i class="fa fa-search"></i></button>
+        
+            <div class="leftnav">
+                    <a href="#" title="Create a video or post"><i class="fa fa-video-camera"></i></a>
+                    <a href="#" title="Youtube apps"><i class="fa fa-th"></i></a>
+                    <a href="#" title="Messages"><i class="fa  fa-envelope"></i></a>
+            </div>
+        </div>
+    </div>
+    <div class="videobox">
```

Comment:
[OPTIONAL]
It's also a good practice to use the `main` tag for the most relevant content of your page.